package.json

```diff
@@ -15,43 +15,43 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^9.1.2",
-    "@angular/cdk": "^9.2.1",
-    "@angular/common": "^9.1.2",
-    "@angular/compiler": "^9.1.2",
-    "@angular/core": "^9.1.2",
-    "@angular/forms": "^9.1.2",
-    "@angular/material": "^9.2.1",
-    "@angular/platform-browser": "^9.1.2",
-    "@angular/platform-browser-dynamic": "^9.1.2",
-    "@angular/router": "^9.1.2",
+    "@angular/animations": "^10.1.0",
+    "@angular/cdk": "^10.2.0",
+    "@angular/common": "^10.1.0",
+    "@angular/compiler": "^10.1.0",
+    "@angular/core": "^10.1.0",
+    "@angular/forms": "^10.1.0",
+    "@angular/material": "^10.2.0",
+    "@angular/platform-browser": "^10.1.0",
+    "@angular/platform-browser-dynamic": "^10.1.0",
+    "@angular/router": "^10.1.0",
     "core-js": "^2.5.4",
     "jdnconvertiblecalendar": "0.0.5",
     "rxjs": "^6.5.5",
-    "tslib": "^1.10.0",
+    "tslib": "^2.0.0",
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.901.1",
-    "@angular-devkit/build-ng-packagr": "~0.901.1",
-    "@angular/cli": "^9.1.1",
-    "@angular/compiler-cli": "^9.1.2",
-    "@angular/language-service": "^9.1.2",
+    "@angular-devkit/build-angular": "~0.1001.0",
+    "@angular-devkit/build-ng-packagr": "~0.1001.0",
+    "@angular/cli": "^10.1.0",
+    "@angular/compiler-cli": "^10.1.0",
+    "@angular/language-service": "^10.1.0",
     "@types/jasmine": "~2.8.6",
     "@types/jasminewd2": "~2.0.3",
     "@types/node": "^12.11.1",
     "codelyzer": "^5.1.2",
```

Comment:
I get the following warning during the install: `npm WARN codelyzer@5.2.2 requires a peer of @angular/compiler@>=2.3.1 <10.0.0 || >9.0.0-beta <10.0.0 || >9.1.0-beta <10.0.0 || >9.2.0-beta <10.0.0 but none is installed. You must install peer dependencies yourself.` 
Version **6.0.0** will fix it.

Reply:
Hm, works for me.

Could you try removing `node_modules` first and then run `npm i`?

Reply:
this is the log from travis:

https://github.com/dhlab-basel/JDNConvertibleCalendarDateAdapter/runs/1149505780

Reply:
I do not have the error when I delete the node mdules, but as soon as I re-run `npm i` I get the error

Reply:
Ok, same for me now.

However, I also get these warnings in dsp-ui when I re-run `npm i`. Honestly: no idea why.

Reply:
I do not get this error anymore when I use `"codelyzer": "^6.0.0"` but there are other warnings and I cannot be sure where they come from

Reply:
these are dev deps, right? So they shouldn't affect production

Reply:
no they should not

Reply:
I checked, and we have the same version of codelyzer in dsp-ui. It was not updated during the Angular migration.

Reply:
ok, I create a DSP task so we can observe this when doing the Angular 10 update in dsp-ui: https://dasch.myjetbrains.com/youtrack/issue/DSP-691